bitbucket-pipelines.yml

```diff
@@ -1,27 +1,122 @@
-image: quay.io/pantheon-public/build-tools-ci:5.x
+# These are yaml anchors, reused later
+x-bash-env-steps:
+  - &bash_env_export export BASH_ENV="$BITBUCKET_CLONE_DIR/.bashrc"
+  - &bash_env_source source $BASH_ENV
 
-pipelines:
-  default:
-    - step:
-        caches:
-          - composer
-        script:
-          - apt-get update && apt-get install -y unzip
-          - curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer
-          - composer install --no-ansi --no-interaction --optimize-autoloader --no-progress
-          - CI_BRANCH=$BITBUCKET_BRANCH CI_BUILD_NUMBER=$BITBUCKET_BUILD_NUMBER PR_NUMBER=$BITBUCKET_PR_ID /build-tools-ci/scripts/set-environment
-          - source ~/.bashrc && composer -n install
+definitions:
+  caches:
+    terminusplugins: ${TERMINUS_PLUGINS_DIR:-~/.terminus/plugins}
```

Comment:
@aaronbauman what's the reason for caching Terminus plugins? They are pre-installed in the Docker image

Reply:
debugging cruft